.github/actions/pull_images/action.yml

```diff
@@ -59,4 +59,4 @@ runs:
 
     - name: Pull
       shell: bash
-      run: docker compose pull
+      run: cat compose.yaml && docker compose pull
```

Comment:
leftover ?

Reply:
I left it on purpose because it shows what's actually going on, but i can remove it

Reply:
nope, that's fine.